eml/components/explore/ExploreCard.jsx

```diff
@@ -1,34 +1,224 @@
-import { View, Text, Image, Pressable } from 'react-native'
-import React from 'react'
-import { useFonts, VarelaRound_400Regular } from '@expo-google-fonts/dev'
-import { useNavigation } from '@react-navigation/native'
-import { AppLoading } from 'expo-app-loading'
+import React, { useEffect } from "react";
+import { View, Text, Image, Pressable } from "react-native";
+import Collapsible from "react-native-collapsible";
+import { useNavigation } from "@react-navigation/native";
+import { useFonts, VarelaRound_400Regular } from "@expo-google-fonts/dev";
+import { AppLoading } from "expo-app-loading";
 
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+
+import CardLabel from "./CardLabel";
+import CustomRating from "./CustomRating";
+import SubscriptionButton from "./SubscriptionButton";
+import AccesCourseButton from "./AccesCourseButton";
+
+import { checkIfSubscribed, subscribeToCourse, ifSubscribed } from "../../api/api";
+
+
+
+
+export default function ExploreCard({ course }) {
+  const [isCollapsed, setIsCollapsed] = React.useState(true);
+  const [isSubscribed, setIsSubscribed] = React.useState(false);
+  const navigation = useNavigation();
+
+  
 
-export default function ExploreCard({ title, courseId }) {
-  const navigation = useNavigation()
   let [fontsLoaded] = useFonts({
-    VarelaRound_400Regular
-  })
+    VarelaRound_400Regular,
```

Comment:
Should probably be removed. Wrong font (leftover from last year, likely)

Reply:
video group ...

Reply:
@Educado-App/video-streaming-team